airbyte-integrations/connectors/source-fullstory/source_fullstory/manifest.yaml

```diff
@@ -0,0 +1,188 @@
+version: "0.29.0"
+
+definitions:
+  requester:
+    url_base: "https://api.fullstory.com"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_key'] }}"
+
+  large_value_filter:
+    type: CustomFilter
+    filter_function: |
+      def filter_records(records, stream_state, config):
+          max_state_value = stream_state.get("createdAt", "9999-04-12T18:13:36.000Z")
+          if max_state_value == "9999-04-12T18:13:36.000Z":
+              return []
+          return records
```

Comment:
Why you're using this function?

Reply:
it was part of a test, and is redundant, removing it.